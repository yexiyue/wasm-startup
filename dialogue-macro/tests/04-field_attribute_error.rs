use dialogue_macro::Dialogue;

#[derive(Dialogue)]
struct User {
    #[dialogue(default1 = "wasm-project", prompt = "请输入项目名称" ,options=["wasm-project","wasm-project2",123], multi_select = true)]
    name: String,
    age: i32,
}

fn main() {}
