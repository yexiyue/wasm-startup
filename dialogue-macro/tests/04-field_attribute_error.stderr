error: only support default, prompt, options
 --> tests/04-field_attribute_error.rs:5:16
  |
5 |     #[dialogue(default1 = "wasm-project", prompt = "请输入项目名称" ,options=["wasm-project","wasm-project2",123], multi_select = true)]    ...
  |                ^^^^^^^^
